README.md

```diff
@@ -123,10 +123,12 @@ custom:
     mappingTemplatesLocation: # defaults to mapping-templates
     mappingTemplates:
       - dataSource: # data source name
-        type: # type name in schema (e.g. Query, Mutation, Subscription)
+        type: # type name in schema (e.g. Query, Mutation, Subscription, PIPELINE)
```

Comment:
@markvp `type` does not take `PIPELINE` as a value, it's under `kind`.

Reply:
oops.  fixed.